Custom Definitions.event

```diff
@@ -1,6 +1,8 @@
 #ifndef VBABT_DEFINITIONS
 #define VBABT_DEFINITIONS
 
+#include "EAStdlib.event"
```

Comment:
```suggestion
#include "EAstdlib.event"
```

filename case reee (TODO: fix this literally everywhere else).

(This will not assemble on Linux)